mde-services/src/main/java/de/terrestris/mde/mde_backend/controller/MetadataCollectionController.java

```diff
@@ -221,12 +226,30 @@ public ResponseEntity<List<String>> getAutomaticKeywords(@PathVariable("metadata
     return new ResponseEntity<>(DatasetIsoGenerator.getAutomaticKeywords(isoMetadata), OK);
   }
 
+  // TODO Make validation canceable?
+  // TODO Add openapi annotations
   @GetMapping("/{metadataId}/validate")
-  public ResponseEntity<List<String>> validate(@PathVariable("metadataId") String metadataId) {
+  public ResponseEntity<String> validate(@PathVariable("metadataId") String metadataId) {
```

Comment:
You'll probably want to keep the list here to be able to format the errors properly in the client

Reply:
The interface got changed to actually start the validation process only, the return value is just the metadata id of the collection the validation got started for.